tests/conftest.py

```diff
@@ -202,6 +202,15 @@ def pytest_collection_modifyitems(session, items):
                     )
                     items.remove(item)
                     continue
+    # skip UI test on openshift dedicated ODF-MS platform
+    if config.ENV_DATA["platform"].lower() == constants.OPENSHIFT_DEDICATED_PLATFORM:
+        for item in items.copy():
+            if "/ui/" in str(item.fspath):
```

Comment:
I think there are UI test cases within test packages that are not called `ui`.
let me check

Reply:
please ignore, we don't have